ofrak_core/CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Show mapped resource captions on hover in the hex view [#221](https://github.com/redballoonsecurity/ofrak/pull/221)
 - Change how resources are stored to making deleting (and thus packing) much faster [#201](https://github.com/redballoonsecurity/ofrak/pull/201)
 
+### Added
+- Add stub `xattr` module to use on platforms without `xattr` support (e.g., Windows) [#228](https://github.com/redballoonsecurity/ofrak/pull/228)
+- Test stub `xattr` module [#228](https://github.com/redballoonsecurity/ofrak/pull/228)
```

Comment:
```suggestion
```

Adding tests is not a feature that users need to know about -- it should be assumed that we test features that exist.